rtl/eclrtl/rtlbcd.hpp

```diff
@@ -82,16 +80,15 @@ ECLRTL_API char  DecClip(void *);
 ECLRTL_API void  DecRoundPos(void *,int by);
 ECLRTL_API void  SetMAccum(unsigned char c);
 
-ECLRTL_API void  DecLock();
-ECLRTL_API void  DecUnlock();
 ECLRTL_API unsigned DecMarkStack();
 ECLRTL_API void DecReleaseStack(unsigned mark);
 
+//No longer a critical section (since stack is thread_local), but prevents problems with exceptions.
```

Comment:
Well, I was going to say "We are targetting 6.0.0, so eclhelper version will be changing, so we are ok to require recompilation, though leaving the stubs in there to ensure they load long enough to give the error makes sense".

But then I wondered if this should not in fact target 5.6.0 (in which case we may want to make sure that old WUs both load and run)